.github/workflows/build.yml

```diff
@@ -28,9 +28,9 @@ jobs:
         run: |
           mkdir -p $HOME/graalvm && cd $HOME/graalvm
           curl -i -u "gluonteam:$PAT" https://api.github.com/repos/graalvm/graalvm-ce-dev-builds/releases > releases.json
-          export GRAALVM_GITHUB_LINUX_AMD64_DOWNLOAD_URL=$(cat releases.json | grep browser_download_url | grep graalvm-ce-java11-linux-amd64-20.3 | head -n 1 | sed -e 's/.*\(https.*\)".*/\1/g')
+          export GRAALVM_GITHUB_LINUX_AMD64_DOWNLOAD_URL=$(cat releases.json | grep browser_download_url | grep graalvm-ce-java11-linux-amd64 | head -n 1 | sed -e 's/.*\(https.*\)".*/\1/g')
           wget ${GRAALVM_GITHUB_LINUX_AMD64_DOWNLOAD_URL}
-          tar -xzvf graalvm-ce-java11-linux-amd64-20.3.0-dev.tar.gz -C $HOME/graalvm && cd $GITHUB_WORKSPACE
+          tar -xzvf graalvm-ce-java11-linux-amd64-dev.tar.gz -C $HOME/graalvm && cd $GITHUB_WORKSPACE
```

Comment:
Does this mean there is a (latest) version without version tag?

Reply:
Yes, but the path to that version contains the full version tag.